certified-connectors/N-able Cloud Commander/readme.md

```diff
@@ -0,0 +1,57 @@
+## Cloud Commander
+Cloud Commander is a unified Microsoft CSP portal that lets you automate common tasks across all Microsoft 365 apps without switching consoles or customers.
```

Comment:
Please add all required fields for readme.md file from the below template.

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md

Reply:
Fixed